apps/qubit/modules/settings/actions/globalAction.class.php

```diff
@@ -364,6 +366,16 @@ protected function updateGlobalSettings()
       $setting->save();
     }
 
+    // Google Maps Javascript API key
+    if (null !== $googleMapsApiKey = $thisForm->getValue('google_maps_api_key'))
+    {
+      $setting = QubitSetting::getByName('google_maps_api_key');
```

Comment:
We can save the migration for `google_maps_api_key` if you create the object on demand, right here, if `$setting` is `NULL`. You did something very similar int he following commit a while ago: f3041d0d291437d5a0816f8c3488b36661e5f1f2.